.gitignore

```diff
@@ -159,4 +159,8 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
```

Comment:
We usually don't push .gitignore to github. it's more for local environment to ignore the files/folders you don't want in github.

Reply:
I have to disagree. The .gitignore is a great file to track in the repository itself. It has benefits such as preventing secrets being committed and clearing outputs generated from running the code itself from clogging up the source control for all potential users.